cmd/main.go

```diff
@@ -61,8 +61,8 @@ func main() {
 		os.Exit(1)
 	}
 
-	additionalCASrc := fmt.Sprintf("%s/trusted-ca.crt", builder.ConfigMapCertsMountPath)
-	additionalCADst := fmt.Sprintf("%s/openshift-trusted-ca.crt", tlsCertRoot)
+	additionalCASrc := fmt.Sprintf("%s/additional-ca.crt", builder.ConfigMapCertsMountPath)
```

Comment:
@deads2k are we better off putting this constant in the API?

Reply:
it's not a user-facing api, it's an internal implementation detail of the controller/builder contract, so i would say no, it does not belong in the api.

Reply:
Don't see too many options other than policing ourselves/e2e testing - this is too simple to be added to`library-go`, and we don't have other shared repos that fit this use case.

Reply:
> Don't see too many options other than policing ourselves/e2e testing - this is too simple to be added to`library-go`, and we don't have other shared repos that fit this use case.

We don't vendor for constants.